Packs/F5LTM/Playbooks/F5_LTM_-_Check_if_any_single_node_pool.yml

```diff
@@ -0,0 +1,286 @@
+id: F5 LTM - Check if any single node pool
+version: -1
+name: F5 LTM - Check if any single node pool
+description: A playbook to check if any pool has one node as a single node.
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: df413feb-a4d3-412b-8293-608c69ee521f
+    type: start
+    task:
+      id: df413feb-a4d3-412b-8293-608c69ee521f
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "3"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 360
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "3":
+    id: "3"
+    taskid: 2fca0f1c-e977-4aea-835a-50c414b47a6c
+    type: regular
+    task:
+      id: 2fca0f1c-e977-4aea-835a-50c414b47a6c
+      version: -1
+      name: Get Pool Members
+      description: Get Pool Members
+      script: '|||f5-ltm-get-pool-members'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "4"
+    scriptarguments:
+      pool:
+        complex:
+          root: F5.LTM.Nodes
+          accessor: pools
+      pool_name:
+        complex:
+          root: inputs.Nodes
+          accessor: pools
+          transformers:
+          - operator: uniq
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 265,
+          "y": 545
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "4":
+    id: "4"
+    taskid: 92e219af-61b4-4fee-8611-a4c7583be088
+    type: condition
+    task:
+      id: 92e219af-61b4-4fee-8611-a4c7583be088
+      version: -1
+      name: Enough Members in the Pool
+      type: condition
+      iscommand: false
+      description: ""
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "8"
+      "yes":
+      - "7"
+    separatecontext: false
+    conditions:
```

Comment:
@ayman-m As discussed, please make sure to change the condition in order to validate the nodes names correctly (I think you can use the `cut` transformer). If necessary, you can add another condition before that to verify it.